experimental/sdk/webpubsub-socketio-extension/src/EIO/components/client-connection-context.ts

```diff
@@ -1,6 +1,63 @@
+import { debugModule } from "../../common/utils";
+import { HubSendToConnectionOptions, WebPubSubServiceClient, JSONTypes} from "@azure/web-pubsub";
+import { RequestBodyType } from '@azure/core-rest-pipeline';
+import { ConnectResponse as WebPubSubConnectResponse, ConnectResponseHandler as WebPubSubConnectResponseHandler } from "@azure/web-pubsub-express";
+import { WEBPUBSUB_CONNECT_RESPONSE_FIELD_NAME } from "./constants";
+
+const debug = debugModule("wps-sio-ext:EIO:ClientConnectionContext");
+
 /**
  * A logical concept that stands for an Engine.IO client connection, every connection has a unique `connectionId`.
  * It maps Engine.IO Transport behaviours to Azure Web PubSub service REST API calls.
  */
-class ClientConnectionContext {
-}
\ No newline at end of file
+export class ClientConnectionContext {
+  public serviceClient: WebPubSubServiceClient;
+  public connectionId: string;
+  public connectResponded: boolean;
+  private _connectResponseHandler: WebPubSubConnectResponseHandler;
+
+  constructor(serviceClient: WebPubSubServiceClient, connectionId: string, connectResponseHandler: WebPubSubConnectResponseHandler) {
+    this.serviceClient = serviceClient ?? (() => { throw new Error("serviceClient cannot be null"); })();
+    this.connectionId = connectionId ?? (() => { throw new Error("serviceClient cannot be null"); })();
+    this._connectResponseHandler = connectResponseHandler?? (() => { throw new Error("connectResponseHandler cannot be null"); })();
+    this.connectResponded = false;
+  }
+
+  /**
+   * Send `message` to a the bound client connection.
+   * @param message The message
+   * @param cb Callback function to handle error
+   */
+  public async send(message: string, cb?: (err?: Error) => void) {
+    debug(`send message ${message}, type = ${typeof message}`);
+
+    var options: HubSendToConnectionOptions = {};
+    options["contentType"] = "text/plain";
```

Comment:
```suggestion
    const options: HubSendToConnectionOptions = { contentType: "text/plain" };
```
can `var` pass eslint?